scripts/cat/cats.py

```diff
@@ -698,38 +699,60 @@ def grief(self, body: bool):
+            levels = rel_with_dead.get_value_levels()
+            for level in levels:
+                if level.is_romance_level():
+                    if level.is_extreme_pos():
+                        very_high_values.append(RelValue.ROMANCE)
+                    elif level.is_low_pos():
+                        high_values.append(RelValue.ROMANCE)
+                    continue
 
-            if to_self.romantic_love > 55:
-                very_high_values.append("romantic")
-            if to_self.romantic_love > 40:
-                high_values.append("romantic")
-
-            if to_self.platonic_like > 50:
-                very_high_values.append("platonic")
-            if to_self.platonic_like > 30:
-                high_values.append("platonic")
+                if level.is_like_level():
+                    if level.is_extreme_pos():
+                        very_high_values.append(RelValue.LIKE)
+                    elif level.is_low_pos():
+                        high_values.append(RelValue.LIKE)
+                    elif level.is_extreme_neg():
+                        very_low_values.append(RelValue.LIKE)
+                    continue
 
-            if to_self.admiration > 70:
-                very_high_values.append("admiration")
-            if to_self.admiration > 50:
-                high_values.append("admiration")
+                if level.is_respect_level():
+                    if level.is_extreme_pos():
+                        very_high_values.append(RelValue.RESPECT)
+                    elif level.is_low_pos():
+                        high_values.append(RelValue.RESPECT)
+                    elif level.is_extreme_neg():
+                        very_low_values.append(RelValue.RESPECT)
+                    continue
 
-            if to_self.comfortable > 60:
-                very_high_values.append("comfort")
-            if to_self.comfortable > 40:
-                high_values.append("comfort")
+                if level.is_trust_level():
+                    if level.is_extreme_pos():
+                        very_high_values.append(RelValue.TRUST)
+                    elif level.is_low_pos():
+                        high_values.append(RelValue.TRUST)
+                    elif level.is_extreme_neg():
+                        very_low_values.append(RelValue.TRUST)
+                    continue
 
-            if to_self.trust > 70:
-                very_high_values.append("trust")
-            if to_self.trust > 50:
-                high_values.append("trust")
+                if level.is_comfort_level():
+                    if level.is_extreme_pos():
+                        very_high_values.append(RelValue.COMFORT)
+                    elif level.is_low_pos():
+                        high_values.append(RelValue.COMFORT)
+                    elif level.is_extreme_neg():
+                        very_low_values.append(RelValue.COMFORT)
+                    continue
```

Comment:
```suggestion
                if level.is_extreme_pos():
                    very_high_values.append(level.get_rel_value)
                elif level.is_low_pos():
                    high_values.append(level.get_rel_value)
                elif level.is_extrem_neg():
                    very_low_values.append(level.get_rel_value)
                continue
```

all operating under the assumption that you add another helper property to `ValueLevel` called `get_rel_value` along the following lines:
```
    def get_rel_value(self):
        if self.is_like_level():
            return RelValue.LIKE
        if self.is_respect_level():
            return RelValue.RESPECT
        if self.is_trust_level():
            return RelValue.TRUST
        if self.is_comfort_level():
            return RelValue.COMFORT
        if self.is_romance_level():
            return RelValue.COMFORT
        raise ValueError(f"ValueLevel {self.name} provided does not belong to any known RelValue!")
```